composer/loggers/wandb_logger.py

```diff
@@ -123,12 +122,10 @@ def log_file_artifact(self, state: State, log_level: LogLevel, artifact_name: st
                 warnings.warn(("WandB permits only alpha-numeric, periods, hyphens, and underscores in artifact names. "
                                f"The artifact with name '{artifact_name}' will be stored as '{new_artifact_name}'."))
 
-            if self._enabled and self._log_artifacts:
-                import wandb
-                extension = new_artifact_name.split(".")[-1]
-                artifact = wandb.Artifact(name=new_artifact_name, type=extension)
-                artifact.add_file(os.path.abspath(file_path))
-                wandb.log_artifact(artifact, aliases=aliases)
+            extension = new_artifact_name.split(".")[-1]
```

Comment:
It looks like the if check and the condition import were removed? Can it be added back in?

Reply:
LMAO; I promise I can read python. Thanks!